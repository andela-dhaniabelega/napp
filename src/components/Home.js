import React from 'react';
import '../../sass/styles.scss';
import App from 'components/app';
import Header from 'components/Header';
import Headline from 'components/Headline';
import Featured from 'components/Featured';


export default class Home extends React.Component {
	render() {
		return (
     <div>
        <App />
        <Header />
        <Headline />
        <Featured />
      </div>
      );
	}
}
