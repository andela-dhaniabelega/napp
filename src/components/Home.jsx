import React from 'react';
import '../../sass/styles.scss';
import Header from '../components/Header';
import Slider from '../components/Slider';
import HomeContents from '../components/HomeContents';
import Footer from '../components/Footer';

const Home = () => (
  <div className="wrapper">
    <div className="center">
      <Header /> 
      <Slider />
      <HomeContents />
      <Footer />
    </div> 
  </div> 
);

export default Home;
