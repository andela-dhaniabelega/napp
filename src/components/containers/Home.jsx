import React from 'react';
import '../../../public/sass/styles.scss';
import Header from '../../components/Header';
import Slider from '../../components/Slider';
import HomeContents from '../containers/HomeContents';
import Footer from '../../components/Footer';
import Bar from '../../components/Bar';

/**
 * Stateless Component
 * It simply renders child components
 * 
 * @return {void}
 */
const Home = () => (
  <div className="wrapper">
    <div className="center">
      <Header /> 
      <Bar />
      <Slider />
      <HomeContents />
      <Footer />
    </div> 
  </div> 
);


export default Home;

