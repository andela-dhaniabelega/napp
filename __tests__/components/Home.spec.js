import React from 'react';
import renderer from 'react-test-renderer'
import Home from '../../src/components/containers/Home';


describe('Home Page', () => {
		it('renders a snapshot', () => {
          const tree = renderer.create(<Home />).toJSON();
          expect(tree).toMatchSnapshot();
		});
});