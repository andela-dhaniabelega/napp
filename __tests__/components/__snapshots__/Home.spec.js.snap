// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Home renders a snapshot 1`] = `
<div
  className="wrapper"
>
  <div
    className="center"
  >
    <div
      className="header"
    >
      <div
        className="logo floatleft"
      >
        <a
          href="/"
        >
          <img
            alt=""
            src="../public/images/logo copy.png"
          />
        </a>
      </div>
      <div
        className="top_menu floatleft"
      >
        <ul>
          <li>
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/about"
            >
              About Napp
            </a>
          </li>
          <li>
            <a
              href="/contact"
            >
              Contact us
            </a>
          </li>
          <li>
            <a
              onClick={[Function]}
              role="button"
            >
              Sign In with Google
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="main_menu_area"
    >
      <ul
        id="nav"
      />
    </div>
    <div
      className="slider_area"
    >
      <div
        className="slider"
      >
        <ul
          className="bxslider"
        >
          <li>
            <img
              alt=""
              src="../public/images/welcome.png"
            />
          </li>
        </ul>
      </div>
    </div>
    <div
      className="content_area"
    >
      <div
        className="welcome"
      >
        <p>
          The Napp Journal provides news headlines from over 78 news sources. These news Sources include CNN, BBC, Al Jazeera, and much more. The App provides a friendly user interface that allows you view major headlines at a glance.
        </p>
        <p>
          You can view all the news sources, review headlines and more. Start exploring now! and thank us later!.. *_*
        </p>
      </div>
    </div>
    <div
      className="footer_bottom_area"
    >
      <div
        className="copyright_text"
      >
        <p>
          Copyright Â© 2017 Napp. All rights reserved
        </p>
      </div>
    </div>
  </div>
</div>
`;
